### ISSUE TITLE
Bug: Teams still shown on dashboard even when deleted/left

### ISSUE THREAD

**Issue by anastasia21112** [open-learning-exchange/myplanet#1875]
## Description
On the user's dashboard, their previous teams are still shown even if they are completely deleted from the community or when the user has left them. 

## Screenshots
![Screen Shot 2020-06-24 at 9 38 14 AM](https://user-images.githubusercontent.com/49795308/85586955-95eed780-b5fe-11ea-981e-8b0ef88a608e.png)
![Screen Shot 2020-06-24 at 9 38 24 AM](https://user-images.githubusercontent.com/49795308/85586976-9a1af500-b5fe-11ea-9a39-7c9aa1ff46c8.png)

## Steps to Recreate
* Create team from admin account on Planet
* Request to join from community member account on myPlanet
* Accept request from admin account on Planet
* Sync myPlanet with Planet
* Delete team from admin account on Planet
* View user dashboard and teams on myPlanet

**Comment by anthonyhvo12** [comment]
Good catch! It seems like If one leaves a team from myPlanet but not on Planet, the left team will appear again on next sync. The user will have to leave/delete the team both on myPlanet and Planet for the app to display properly.

**Comment by Okuro3499** [comment]
No longer the case

https://github.com/open-learning-exchange/myplanet/assets/64019708/c95ff88f-c51f-4596-a5ed-56573b737039
